runhouse/servers/cluster_servlet.py

```diff
@@ -407,8 +408,16 @@ def status(self):
     ##############################################
     def _get_logs(self):
         with open(SERVER_LOGFILE) as log_file:
-            log_lines = log_file.readlines()
-            return " ".join(log_lines)
+            log_lines = "".join(log_file.readlines())
+            chars_to_remove_from_logs = {
```

Comment:
this looks a little hacky, what's going in with the logs here that we need to be cleaning it in this way?